### ISSUE TITLE
Dark Mode / Light Mode

### ISSUE THREAD

**Issue by juliaxbow** [pln-planning-tools/MapLight#76]
I'll add in color guide here

**Comment by wilkyr31d** [comment]
Tagging post-MVP. Yell if you disagree @juliaxbow

**Comment by juliaxbow** [comment]
Agreed, though we will need to decide whether to go dark or light for MVP! Either way I'll have color guides

**Comment by wilkyr31d** [comment]
I think light.
Thinking about it more, should it really just be a binary choice, or should it all be customizable?

**Comment by SgtPooki** [comment]
> I think light. Thinking about it more, should it really just be a binary choice, or should it all be customizable?

dark/light mode is pretty standard.. "all be customizable" is itself a discussion that could take a week to nail down well

**Comment by wilkyr31d** [comment]
I mean, yeah... but I don't know if dark/light mode is really that relevant to the core use cases here. It's "sugar topping", but nothing more. What the user MIGHT actually want is the ability to control the way the roadmap looks by adjusting colors. That seems much more valuable to real world usage than dark/light. That said, 100% agree that it's a swampy topic and does not need to be solved now. I think right now keeping it light is inline with all the renders I went over with Juan; switching to a dark mode seems like a big jump, so let's just keep it light.

**Comment by juliaxbow** [comment]
Color guides (both dark and light) are in Figma [here](https://www.figma.com/file/7l5BAnM9ZDxDJePm3yIJFS/Roadmapping-Tool-MVP?node-id=460%3A4403&t=HsU3cgBh6cQU1CBx-0)

**Comment by juliaxbow** [comment]
For post-MVP different color themes (so not fully customizable but at least giving a handful of options for color themes) would be pretty cool
